lib/azure/resource_management/ARM_deployment_template.rb

```diff
@@ -33,6 +33,11 @@ def ohai_hints(hint_names, resource_ids)
         end
       end
 
+      if @platform == 'Windows'
+        hints_json['public_winrm_port'] = 5985

```

Comment:
isn't this the default winrm port for unencrypted winrm and 5986 is the default port for https? this is always 5985?